setup.cfg

```diff
@@ -6,6 +6,7 @@ ignore_missing_imports = True
 warn_unused_configs = True
 warn_redundant_casts = True
 warn_unused_ignores = True
+plugins = idom.mypy
```

Comment:
We need to document this somehow.

Undocumented features essentially do not exist.